arbitrum-docs/useful-addresses.mdx

```diff
@@ -1,8 +1,9 @@
 import { AddressExplorerLink as AEL } from '@site/src/components/AddressExplorerLink'
 
-# Arbitrum useful Addresses
+# Useful Arbitrum contract addresses
+
+The following contract addresses that may be useful to those building on Arbitrum:
```

Comment:
A potential future optimization: describe when/why each of these would be useful.